app/code/Magento/CheckoutAgreements/etc/webapi.xml

```diff
@@ -13,4 +13,10 @@
             <resource ref="Magento_Sales::sales" />
         </resources>
     </route>
+    <route url="/V1/carts/licence/listing" method="GET">
```

Comment:
I would suggest renaming route to `list`. 
Please refer to `Magento\Catalog\Api\CategoryListInterface` to remain consistent with it.